packages/launcher/src/main/rpc/app.js

```diff
@@ -408,6 +408,32 @@ export const sandboxed = {
       }
     },
   },
+
+  storage_delete: {
+    params: {
+      key: STORAGE_KEY_PARAM,
+    },
+    handler: async (
+      ctx: AppContext,
+      params: { key: string },
+    ): Promise<void> => {
+      try {
+        const { feedHash, manifestHash } = await getStorageManifestHash(ctx)
+        const newManifestHash = await ctx.bzz.deleteResource(
```

Comment:
This is duplicated with the `dataHash` created in the `Promise.all()` below, this should be removed

Reply:
Nice catch, didn't notice that I left it like that when I was changing the way I was deleting the resource :+1: